### PR TITLE
Adds a prop that keeps the DayPicker open when you select a date

### PR DESCRIPTION
**Pull request by majapw**
to: @moonboots @airbnb/webinfra 

The prop needs a better name. Also this needs tests. But I'm literally dying.

However, this behavior was pretty easy to implement so that's cool.

### CODE REVIEW

**Comment by moonboots** [comment]
lgtm thanks!

File: src/components/DateRangePicker.jsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ const defaultProps = {
   disabled: false,
   required: false,
   reopenPickerOnClearDates: false,
+  keepOpen: false,
```
Comment (ljharb) [inline src/components/DateRangePicker.jsx]: maybe `keepOpenOnSelect`?

File: stories/SingleDatePicker.js [hunk 2]
```diff
@@ -70,12 +70,17 @@ storiesOf('SingleDatePicker', module)
       showClearDate
     />
   ))
-  .add('with clear dates button (Picker Displayed)', () => (
+  .add('eopens DayPicker on clear dates', () => (
```
Comment (ljharb) [inline stories/SingleDatePicker.js]: `s/eopens/reopens`

**Comment by majapw** [comment]
Addressed comments! PTAL @ljharb

**Comment by majapw** [comment]
This also fixes https://github.com/airbnb/react-dates/issues/62

**Pull request merged**